src/style.css

```diff
@@ -0,0 +1,120 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+}
+
+.card__item {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin: auto;
+  font-size: 12px;
+  width: 166px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  margin: auto;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  width: 166px;
+  color: #616070;
+}
+
+.card__reviews {
+  padding-top: 16px;
+  margin: auto;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  width: 63px;
+  padding-left: 16px;
+
```

Comment:
do not hardcode elements width - you have already done it for the whole card
so use width: 100% here